foundations/html_css/css-foundations/intro-to-css.md

```diff
@@ -80,6 +80,8 @@ Class selectors will select all elements with the given class, which is just an
 ```
 
 Note the syntax for class selectors: a period immediately followed by the case-sensitive value of the class attribute. Classes aren't required to be specific to a particular element, so you can use the same class on as many elements as you want.
+>The selector will not work if the class name begins with a digit, e.g. `.4lert-text` will not select an element even if it has the class `4lert-text` - for now, just avoid leading digits in class selector names.
```

Comment:
We use specific markup to render note boxes on the site - you can find the details in the [notes section](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#note-boxes) of our style guide.